lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -208,8 +208,12 @@ function dosomething_global_form_node_form_alter(&$form, &$form_state, $form_id)
   if (!(in_array($form['type']['#value'], $translatable_types))) {
     return;
   }
+
   // Get the last item in the url, to get the edit/add node language.
   $node_lang = array_pop(explode('/', (parse_url(current_path(), PHP_URL_PATH))));
+  if ($node_lang == 'edit') {
```

Comment:
the most pedantic comment, but since it isn't always going to be a language, can we call the var something else. 
`$url_path`?